core/controllers/learner_dashboard.py

```diff
@@ -32,19 +31,12 @@ class LearnerDashboardPage(base.BaseHandler):
 
     @acl_decorators.can_access_learner_dashboard
     def get(self):
-        if self.username in config_domain.BANNED_USERNAMES.value:
```

Comment:
A couple of asides:
(a) When are we getting rid of the obsolete config_domain properties?
(b) When are we turning on the "update role" functionality in the admin page? Note that the next release cut is this weekend.